intro_to_r/intro_to_r.Rmd

```diff
@@ -58,14 +58,17 @@ library(oilabs)
 library(dplyr)
 library(ggplot2)
 ```
-and finally, load in some data:
+
+
```

Comment:
Moved data loading to after section header to distinguish the package installation/loading stage, to the data loading stage.

Rant: RStudio's "promise" of data after the `data(arbuthnot)` command really annoys me and trips up students. Why should they have to run `arbuthnot` first before they can view the data in the Viewer panel?